### PR TITLE
[FR] Cleanup `Play_style`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for cleanup the French translation of the wiki page `Play_style`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Play_style/fr.md
Comment (Walavouchey) [inline wiki/Play_style/fr.md:21]: i see you've done a search and replace, but that broke these links

```suggestion
Ce style de jeu était considéré comme une blague par la plupart des joueurs jusqu'en mai 2013, lorsque le joueur canadien ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578) a commencé à obtenir les meilleurs scores sur des beatmaps très respectées comme [Hatsune Miku - With a Dance Number (val0108)](https://osu.ppy.sh/beatmapsets/29691) et [The Quick Brown Fox - The Big Black (Blue Dragon)](https://osu.ppy.sh/beatmapsets/41823) en jouant avec deux souris.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Play_style/fr.md
Comment (DoryanR) [reply wiki/Play_style/fr.md:21]: Oops, I hadn't seen that, thanks!

**Review by Biokax — CHANGES_REQUESTED** [review]

**Comment by Biokax** [comment]
wait for @Ganondorf-1 review before merging

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:41]:
```suggestion
Les joueurs sur tablette font passer un stylet au-dessus d'une tablette graphique pour naviguer avec le curseur et ne cliquent sur les objets qu'en tapant avec le stylet sur la surface de la tablette graphique elle-même.
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:47]:
```suggestion
La communauté considère qu'il est pratiquement impossible de réaliser des streams en continu avec une seule entrée tactile à un BPM élevé. Ainsi, pour faire des streams en continu en tant que joueur sur tablette uniquement, un joueur doit alterner entre le tapotement de son stylet sur la tablette et l'appui sur une touche secondaire assignée (généralement une touche de la tablette ou du clavier). Cette méthode est parfois appelée *tapx*.
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:49]:
```suggestion
*Remarque : Comme le stylet est considéré comme `M1`, seul le bouton de clic secondaire fonctionnera, et `X` est généralement le bouton de clic secondaire par défaut.*
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:57]:
```suggestion
Les joueurs munis d'une tablette ou d'un clavier font "hover" (survoler) ou "drag" (font glisser) le stylet de la tablette pour naviguer avec le curseur et tapent à l'aide du clavier ou des boutons de la tablette.
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:81]:
```suggestion
- **Plaque métallique** : Il s'agit d'une méthode spéciale qui remplace le clavier pour taper. La méthode a été introduite par ![][flag_US] [geckogates](https://osu.ppy.sh/users/geckogates) et nécessite un peu de câblage et de connaissances électroniques pour la construire. Le joueur attachera un fil aux doigts qui cliquent (connectés au circuit du clavier pour `Z` et `X`) et touchera les plaques de métal pour enregistrer les battements.
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:82]:
```suggestion
- **Manette de jeu** : Tout comme le joystick, cette méthode est considérée comme une façon créative et originale de jouer à osu!. Selon la marque de la manette, le joueur peut utiliser un trackpad intégré, un stick de contrôle/analogique ou un D-Pad pour contrôler le curseur. Certains joueurs trouveront plus confortable d'utiliser les boutons déjà présents sur la manette pour cliquer, ou d'utiliser un clavier ordinaire.
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:98]:
```suggestion
La manette taiko est le style de jeu et la manette par défaut de la série [Taiko no Tatsujin](https://fr.wikipedia.org/wiki/Taiko_no_Tatsujin) ([Taiko Drum Master](https://wikipedia.org/wiki/Taiko:_Drum_Master) pour les États-Unis).
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:120]:
```suggestion
Le compteur de clics montre que les flèches sont saisies comme `L` et `R` et le bouton du dash comme `D`.
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:124]:
```suggestion
Le clavier et la souris sont un style de jeu alternatif mineur du clavier, dans lequel les joueurs peuvent effectuer des dashs en utilisant des clics de souris à la place. Bien que certains joueurs préfèrent l'inverse et préfèrent contrôler les mouvements de gauche et de droite en utilisant les boutons gauche/droit de la souris et activer les dashs avec le clavier.
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:142]:
```suggestion
- **Souris uniquement** : Jouer uniquement à la souris est un style de jeu dans lequel les joueurs contrôlent tous les mouvements et les dashs du catcher uniquement par le biais de la souris. Cette option n'est toutefois disponible que sur les souris d'ordinateur dotées des boutons M4 et M5 (qui se trouvent généralement sur le côté gauche de la souris).
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:33]:
```suggestion
*Note : Le compteur de clics montre les pressions du clavier comme `K1` et `K2`.*
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:43]:
```suggestion
*Remarque : Le compteur de clics montrera toutes les touches de la tablette comme `M1`.*
```

File: wiki/Play_style/fr.md
Comment (Ganondorf-1) [inline wiki/Play_style/fr.md:71]:
```suggestion
*Remarque : Le compteur de clics indique l'entrée tactile comme `M1`.*
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Biokax — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**
